mava/components/jax/building/environments.py

```diff
@@ -93,8 +93,9 @@ def on_building_executor_environment(self, builder: SystemBuilder) -> None:
         Args:
             builder : _description_
         """
-        builder.store.executor_environment = self.config.environment_factory(
-            evaluation=False
+        # Global config set by EnvironmentSpec component
+        builder.store.executor_environment = (
+            builder.store.global_config.environment_factory(evaluation=False)
```

Comment:
small thing but please update mocks.py to also make use of global config

Reply:
Done. Well spotted.